app/models/plate/creator.rb

```diff
@@ -81,8 +93,13 @@ def create_child_plates_from(plate, current_user)
             well.clone.tap do |child_well|
               child_well.aliquots = well.aliquots.map(&:clone)
               child_well.stock_wells.attach(stock_well_picker.call(well))
+
+              creator_parameters.set_well_parameters(child_well) unless creator_parameters.nil?
```

Comment:
Probably should either be removed, or added to the single plate creator bit as well.